packages/kolme/src/submitter/mod.rs

```diff
@@ -58,10 +59,14 @@ impl<App: KolmeApp> Submitter<App> {
     pub fn new_solana(
         kolme: Kolme<App>,
         keypair: kolme_solana_bridge_client::keypair::Keypair,
+        fee_per_cu: Option<u64>,
```

Comment:
Instead of passing optional value, do you think we can have something like a builder pattern, where we do `set_fee_per_cu` instead ?